From 671697cf071eb9259f5d0b91f4d397c14667e7f6 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 8 Dec 2022 14:50:59 +0100
Subject: [PATCH 405/529] ARM: configs: enable low power clocksource on
 fragment-02-multiv7_addons

Enable low power clocksource in STM32MP1 addons fragment.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Ib95dbed3a95f07399874f68f58eac40db8d310cd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283053
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -5,6 +5,7 @@ CONFIG_ARM_SCMI_PROTOCOL=y
 CONFIG_ARM_SMC_MBOX=y
 CONFIG_ARM_STM32_CPUIDLE=y
 CONFIG_CHR_DEV_SG=y
+CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COMMON_CLK_SCMI=y
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_IDLE=y
